update VS Code for the Web - Azure docs to reflect Ignite releases + latest changes (#9062)
Microsoft/vscode-docs: docs/azure/vscodeforweb.md

---
# VS Code for the Web - Azure

VS Code for the Web is a zero-install and browser-based version of Visual Studio Code. The `/azure` (for short) environment, accessible via <https://vscode.dev/azure>, is a dedicated space for Azure development, allowing you to run, debug, and deploy applications to Azure in seconds.

Powered by [Azure Cloud Shell](https://learn.microsoft.com/en-us/azure/cloud-shell/overview), this environment offers up to **4 hours of compute time**, eliminating the need to manually configure dev environments or install dependencies. `/azure` comes preloaded with the latest libraries, extensions, and tools to get you coding instantly.

> ![VS Code for the Web Azure](./images/vscodeforweb/azurescreenshot.png)

Author: I created a PR to add the image. Thought i did it earlier but i guess not

## Azure entry points

> The `/azure` experience integrates with Microsoft Foundry to bring code closer to developers. Buttons like **Open in VS Code for the Web** are available directly within environments like the **Chat Playground** and **Agent Playground** and the **Home Page**. See more in the sample use cases or scenarios section.

Reviewer: ```suggestion
The `/azure` experience integrates with Microsoft Foundry to bring code closer to developers. Buttons like **Open in VS Code for the Web** are available directly within environments like the **Chat Playground** and **Agent Playground** and the **Microsoft Foundry home page**. See more in the sample use cases or scenarios section.
```

3. Select **View Code**, and then select your programming language and SDK.
4. Launch directly into VS Code using the **Open in VS Code** button for the Web with **one click**.

Alternatively, you can create an agent from Microsoft Foundry Home Page (more details below):

> 1. Visit the Homepage for Microsoft Foundry (https://ai.azure.com)

Reviewer: ```suggestion
1. Open the Microsoft Foundry home page (<https://ai.azure.com>)
```

3. Select **Open in VS Code** from the dropdown menu
4. Launch directly into VS Code and use GitHub Copilot to answer any questions you might have.

> We also have integrations with the Azure Portal. Developers can now access an "Open in VS Code" button from Azure Copilot when they generate code.

Reviewer: ```suggestion
We also have integrations with the Azure portal. Developers can now access an "Open in VS Code" button from Azure Copilot when they generate code.
```

To get started:

> 1. Open the [Azure portal](portal.azure.com) and sign in with your Azure account

Reviewer: ```suggestion
1. Open the [Azure portal](https://portal.azure.com) and sign in with your Azure account
```

2. Navigate to Azure Copilot and start developing a scenario you'd like to build
3. Once Copilot has generated code, click on the code file generated and **Open in VS Code**
4. Launch directly in VS Code and use GitHub Copilot to answer any further questions.

## Sample use cases or scenarios

The following are commonly used scenarios for the `\azure` environment.

* [Using Microsoft Foundry to create an agent](https://ai.azure.com)

    1. Access the Microsoft Foundry NextGen portal and select **Open in VS Code** next to the code generated for your agent

>     2. Let the VS Code for the Web - Azure environment initialize and setup your environment

Reviewer: ```suggestion
    2. Let the VS Code for the Web - Azure environment initialize and set up your environment
```

### Account type prompt issue

> If you see a prompt asking "What type of account did you use to start this tunnel?" then it's likely that you're on vscode.dev/azure and not vscode.dev/azure. To fix this, navigate to <https://vscode.dev/azure>.

Reviewer: This is confusing because you mention the same link twice, namely vscode.dev/azure. Should one of these be insiders.vscode.dev/azure?
